### PR TITLE
feat(qc_metrics): Refactor parsing of qc_metrics

### PR DESCRIPTION
**Pull request by henrikstranneheim**
This PR adds/fixes:

- Refactor parsing of qc_metrics
  - Skip unnecessary level parsing
  - Add dispatch and set functions
- Added tests

### How to prepare for test
- [x] ssh to hasta
- [x] activate stage: `us`
- [x] request trailblazer-stage on hasta: `paxa`
- [x] install on stage:
`bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-trailblazer-stage.sh parse_qcmetrics`

### How to test
- [x] login to hasta
- [x] do `us`
- [x] Copy a fresh statusdb to stage and add the yaml files for the run to the stage HK bundle
1. `cg upload delivery-report -f -p quicksnail`
- [x] Remove the analysis record for MIP version 8.2 for quicksnail in statusdb
1. `cg upload delivery-report -f-p quicksnail`

### Expected test outcome
1. check that a delivery report can be created for MIP version 8.2 with missing values: "target_coverage, target_completeness", which are taken from the chanjo database and trailblazer and MIP ✅ 
1. check that a delivery report can be created for MIP version 6.0 with missing values: "target_coverage, target_completeness", which are taken from the chanjo database and trailblazer and MIP ✅ 

## Review
- [ ] code approved by
- [x] tests executed by HS
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a
- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [x] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

**Comment by henrikstranneheim** [comment]
### How to test
- [x] login to hasta
- [x] do `us`
- [x] Copy a fresh statusdb to stage and add the yaml files for the run to the stage HK bundle
1. `cg upload delivery-report -f -p quicksnail`
- [x] Remove the analysis record for MIP version 8.2 for quicksnail in statusdb
1. `cg upload delivery-report -f-p quicksnail`

### Expected test outcome
1. check that a delivery report can be created for MIP version 8.2 with missing values: "target_coverage, target_completeness", which are taken from the chanjo database and trailblazer and MIP ✅ 
Screenshot 2020-08-27 at 10 53 19

1. check that a delivery report can be created for MIP version 6.0 with missing values: "target_coverage, target_completeness", which are taken from the chanjo database and trailblazer and MIP ✅ 
Screenshot 2020-08-27 at 10 47 17
Screenshot 2020-08-27 at 10 50 22

File: tests/mip/test_mip_files.py [hunk 1]
```diff
@@ -5,6 +5,26 @@
 
 from trailblazer.mip import files
 
+
+FATHER_AT_DROPOUT = 2.673848
```
Comment (patrikgrenfeldt) [inline tests/mip/test_mip_files.py:9]: I think these could go into the conftest as a fixture

**Review by patrikgrenfeldt — APPROVED** [review]
Great work!

File: tests/mip/test_mip_files.py (hunk 1 above)
Comment (henrikstranneheim) [reply tests/mip/test_mip_files.py]: Good point! Done

**Comment by henrikstranneheim** [comment]
Deployed on hasta:
Screenshot 2020-08-27 at 13 14 18

**Pull request merged**